app/views/softdrinksindustrylevy/helpers/declaration_details_litres.scala.html

```diff
@@ -40,7 +40,7 @@
     <td>
     @rowData.atHighRate
     </td>
-    <td class="change-answer">
+    <td class="change-answer" onclick="ga('send', 'event', 'declaration', 'clickChange', '@changeRoute')">
```

Comment:
shouldn't the `onclick` attribute be on the `<a>`, not the `<td>`?

Reply:
Yes it should although it still works where it is but will change for consistency